ml-notebook/binder/environment.yml

```diff
@@ -0,0 +1,79 @@
+# packages in addition to repo2docker/buildpacks/conda/environment.yml
+name: pangeo
+channels:
+  - defaults
```

Comment:
This is a great start. I'm not 100% sure how this will work but we may be able to list these in order of preference, something like

```yaml
channels:
  - rapidsai
  - nvidia
  - numba
  - conda-forge
  - anaconda
dependences
  - tensorflow-gpu
  - cupy
  - cudf
  - cuml
...
```

Pinging our conda experts (@ocefpaf and @rsignell-usgs) and GPU experts (@mrocklin and @jacobtomlinson) for some help on the design here.

Reply:
hoping @ocefpaf will comment.  He's the real conda expert.